CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+[[_TOC_]]
```

Comment:
Github automatically generates a table of contents so no need for `[[_TOC_]]` here. LGTM other than that